frontends/verilog/preproc.cc

```diff
@@ -591,7 +591,11 @@ read_define_args()
 
 		default:
 			// The only FSM states are 0-2 and we dealt with 2 at the start of the loop.
-			__builtin_unreachable();
```

Comment:
Just make this `log_assert(false);` instead of this mess.

Reply:
I have applied your suggestion.